host/tests/L3_TestCases/dsFPD/dsFPD_test02_SetVerifyLEDIndicatorsBrightness.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
+#** *****************************************************************************
+# *
+# * If not stated otherwise in this file or this component's LICENSE file the
+# * following copyright and licenses apply:
+# *
+# * Copyright 2024 RDK Management
+# *
+# * Licensed under the Apache License, Version 2.0 (the "License");
+# * you may not use this file except in compliance with the License.
+# * You may obtain a copy of the License at
+# *
+# *
+# http://www.apache.org/licenses/LICENSE-2.0
+# *
+# * Unless required by applicable law or agreed to in writing, software
+# * distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+# *
+#* ******************************************************************************
+
+import os
+import sys
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+sys.path.append(os.path.join(dir_path, "../../"))
+
+from L3_TestCases.dsFPD.dsFPDHelperClass import dsFPDHelperClass
+from dsClasses.dsFPD import dsFPDState
+
+
+class dsFPD_test02_SetVerifyLEDIndicatorsBrightness(dsFPDHelperClass):
+
+    def __init__(self):
+        """
+        Initializes the test02_SetVerifyLEDIndicatorsBrightness test .
+
+        Args:
+            None.
+        """
+        self.testName  = "test02_SetVerifyLEDIndicatorsBrightness"
+        super().__init__(self.testName, '2')
+
+
+    #TODO: Current version supports only manual verification.
+    def testVerifyIndicatorBrightness(self, indicator, brightness, manual=False):
+        """
+        Verifies whether the indicator is showing in teh specified brightness.
+
+        Args:
+            indicator (str) : front panel indicator
+            state (str) : state of the indicator
+            manual (bool, optional): Manual verification (True: manual, False: automated).
+                                     Defaults to False
+
+        Returns:
+            bool : Returns the status of the indicator brightness.
+        """
+        if manual == True:
+            return self.testUserResponse.getUserYN(f"Is {indicator} brightness {brightness}%? (Y/N):")
+        else :
+            #todo: add automation verification methods
+            return False
+    def testFunction(self):
+        """tests the brightness of the LED indicator.
+
+        Returns:
+            bool: final result of the test.
+        """
+
+        self.log.testStart(self.testName, '1')
```

Comment:
This number has to be the test case number, here 2 for test02

Reply:
updated